src/amo/components/Addon/styles.scss

```diff
@@ -229,10 +242,27 @@
 
   .Addon-overall-rating {
     grid-column: 1 / 2;
+
+    .Addon--has-more-addons.Addon-persona & {
+      grid-row: 1 / span 2;
+    }
+
+    .Addon--has-many-more-addons.Addon-persona & {
+      grid-row: 1;
+    }
   }
 
   .AddonMoreInfo {
     grid-column: 1 / 2;
+    grid-row: 4 / span 1;
+
+    .Addon--has-more-addons.Addon-persona & {
+      grid-row: 3 / span 1;
+    }
+
+    .Addon--has-many-more-addons.Addon-persona & {
```

Comment:
Yeah seeing this in the CSS a lot I'm definitely convinced we should rename this class. This is weird to read, the numbers should help I think.